src/MaxMin.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Scanner;
+
+public class MaxMin {
+	
+	void maximum(int[] input) {
+		int max = input[0];
+		for(int index=0; index<input.length; index++) {
```

Comment:
input[0] is already in max. you should start loop from index 1

Reply:
Changes Incorporated.